app/src/main/java/com/github/se/icebreakrr/ui/profile/OtherProfileView.kt

```diff
@@ -259,11 +270,35 @@ fun BottomSheet(aiState: AiViewModel.UiState, onDismissRequest: () -> Unit) {
               when (aiState) {
                 is AiViewModel.UiState.Success -> {
                   Text(
-                      text = "\"${aiState.data}\"",
+                      text = "«${aiState.data}»",
                       fontWeight = FontWeight.Normal,
                       fontSize = TextUnit(CONTENT_FONT_SIZE, TextUnitType.Sp),
                       lineHeight = TextUnit(CONTENT_LINE_HEIGHT, TextUnitType.Sp),
                       modifier = Modifier.testTag("aiResponse"))
+
+                  Spacer(modifier = Modifier.height(30.dp))
+
+                  Row(
+                      horizontalArrangement = Arrangement.Center,
+                      modifier = Modifier.fillMaxWidth()) {
+                        Button(
+                            onClick = onAiRetry,
+                            elevation = ButtonDefaults.buttonElevation(BUTTONS_ELEVATION),
+                            modifier = Modifier.testTag("aiRetry")) {
+                              Icon(imageVector = Icons.Filled.Refresh, contentDescription = "Retry")
+                            }
+
+                        Spacer(modifier = Modifier.width(20.dp))
```

Comment:
Again, a magic number! No magic allowed here! 😠

Reply:
![pandasproblems-the-office](https://github.com/user-attachments/assets/22c57709-7414-41ac-9930-f83004c5fd71)
I hope it's the last one of my career